containers.md

```diff
@@ -33,6 +33,7 @@ We have compiled a list of popular software within the container ecosystem that
 | Envoy	| https://www.envoyproxy.io/docs/envoy/v1.18.3/start/docker ||
 | Tensorflow | https://hub.docker.com/r/armswdev/tensorflow-arm-neoverse |  |
 | Tensorflow serving | 763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-inference-graviton:2.7.0-cpu-py38-ubuntu20.04-e3-v1.0 ||
+| PyTorch | https://hub.docker.com/r/armswdev/pytorch-arm-neoverse ||
```

Comment:
There is no 'latest' tag for pytorch docker image, we need to explicitly specify it. Currently we see better performance with OpenBLAS, so let's use this tag: r22.05-torch-1.10.0-openblas.

Reply:
Added the suggestion as a comment in the table. Will that help?